dashboard/app/jobs.go

```diff
@@ -237,7 +237,6 @@ func createBisectJob(c context.Context, managers map[string]bool, reproLevel das
 		Filter("Status=", BugStatusOpen).
 		Filter("FirstTime>", time.Time{}).
 		Filter("ReproLevel=", reproLevel).
-		Filter("BisectCause=", BisectNot).
```

Comment:
I afraid this will get the query too imprecise. Consider, we select only 300 bugs. So once we bisect 300 bugs, we will select the same bugs again and again and all of them already have bug.BisectCause/Fix != BisectNot, so we will not do any bisection anymore and will never select older bugs.
I think we need to split this into 2 separate queries: 1 for cause and 1 for fix. We do this already for reproLevel, see the comment on the call site.
I think we should also prioritize BisectCause over BisectFix. Splitting the query will allow us to do this easily.

Reply:
> I afraid this will get the query too imprecise. Consider, we select only 300 bugs. So once we bisect 300 bugs, we will select the same bugs again and again and all of them already have bug.BisectCause/Fix != BisectNot, so we will not do any bisection anymore and will never select older bugs.
> I think we need to split this into 2 separate queries: 1 for cause and 1 for fix. We do this already for reproLevel, see the comment on the call site.
> I think we should also prioritize BisectCause over BisectFix. Splitting the query will allow us to do this easily.

Ah yes, I'll fix the incorrect query and prioritize BisectCause, thanks!

Reply:
I've made some changes around how the bugs are fetched, now the priority is as follows.
- BisectCause + C repro
- BisectFix + C repro
- BisectCause + Syz repro
- BisectFix + Syz repro